Formula/n/newnode.rb

```diff
@@ -0,0 +1,70 @@
+class Newnode < Formula
+  desc "Web proxy that uses a distributed p2p network to circumvent censorship"
+  homepage "https://www.newnode.com/newnode-vpn"
+  url "https://github.com/clostra/newnode.git",
+    tag: "2.1.5", revision: "18a3f713267b2a08e34cbe04253df891889997a2"
+  license "GPL-2.0-only"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "coreutils" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+
+  on_macos do
+    depends_on xcode: ["9.3", :build]
+  end
+
+  on_linux do
+    depends_on "gnu-sed" => :build
+    depends_on "grep" => :build
+    depends_on "llvm" => :build
+    depends_on "wget"
+  end
+
+  def install
+    if OS.linux?
+      inreplace "https_wget.c", "/usr/bin/wget", "#{Formula["wget"].opt_bin}/wget"
+      ENV["SED"] = "#{Formula["gnu-sed"].opt_bin}/sed"
+      ENV["GREP"] = "#{Formula["grep"].opt_bin}/grep"
+      ENV["HOMEBREW_CC"] = "#{Formula["llvm"].opt_bin}/clang"
+      ENV["HOMEBREW_CXX"] = "#{Formula["llvm"].opt_bin}/clang++"
```

Comment:
Doesn't it work with GCC?

Reply:
No, it requires clang. GCC doesn't support blocks.

Reply:
That's a lot nicer, thank you